### DISCUSSION TITLE
Microphone passthrough

### DISCUSSION THREAD

**Discussion by TerminatedGB** [ClassicOldSong/Apollo#106] [General]
Will it be possible to implement a microphone passthrough like the one in Parsec? Thank you so much and keep up the good work!!

**Comment by ClassicOldSong** [comment]
Virtual audio driver is way more complicated to implement than virtual display. I don't have that much time to write one recently.

For now you can use some third party remote microphone apps or use VirtualHere with a USB BT dongle, then connect your headphones to that dongle. USB DACs don't work well with VirtualHere but audio over BT over VirtualHere is surprisingly usable. With Alternative A2DP driver you can even get LDAC and Aptx Low Latency coming out directly from Windows.

**Comment by TerminatedGB** [comment]
Okay thank you so much! Appreciate the response~
